.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.0"
```

Comment:
Does this mean next release is gonna be a major, or is this actually fine to skip over?

Reply:
Should probably be a major release, yes.